mlflow/genai/evaluation/session_utils.py

```diff
@@ -77,67 +88,66 @@ def get_first_trace_in_session(session_items: list["EvalItem"]) -> "EvalItem":
 
 
 def evaluate_session_level_scorers(
+    session_id: str,
+    session_items: list["EvalItem"],
     multi_turn_scorers: list[Scorer],
-    session_groups: dict[str, list["EvalItem"]],
 ) -> dict[str, list[Feedback]]:
     """
-    Evaluate multi-turn scorers on grouped sessions.
+    Evaluate all multi-turn scorers for a single session in parallel.
```

Comment:
```suggestion
    Evaluate all multi-turn scorers for a single session.
```
super minor nit to remove implementation detail from docstring